app/models/organization.rb

```diff
@@ -15,6 +15,13 @@ class Organization < ActiveRecord::Base
 
   paginates_per 20
 
+  include PgSearch
+    pg_search_scope :search,
+      :against => :name,
+      :using => {
+        :tsearch => {:prefix => true}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.